### DISCUSSION TITLE
Oening Two Github Account

### DISCUSSION THREAD

**Discussion by MoshTech64** [community/community#29246] [Other Feature Feedback, Questions, & Ideas] [answered]
please, can I own two github account without affect my workflow? And how can I go about it?

**Comment by 0xMRTT** [comment, marked as answer]
You can own only one free account and one machine account

**Reply by MoshTeach64** [reply]
Ok! Thank you for the enlighten. Please,how can I know my one free account and my one machine account. Because I have already opened many account with GitHub, infact I have forgotten what account my machine recognized. Please,put me through.

**Reply by 0xMRTT** [reply]
There is no distinction between machine account and main account. You choose which one is the main account and which one is the machine account. Following the GitHub rules. You should only have one free account and one free account used as machine account for automation

**Reply by MoshTech64** [reply]
okay thank you very much

**Comment by MoshTech64** [comment]
Okay! Thank you for the enlighten. But I have created more than one
account. How should I go about it in other for me to know my first account
and machine account that I can always work with without having issues?

**Reply by 0xMRTT** [reply]
Yes, no problems

**Reply by airtower-luna** [reply]
The important thing is to pick which account is which (yours/machine), and then use the machine account *only* for automation. Also close any additional accounts you might have. Otherwise you risk getting locked out for violating the ToS.

**Comment by kolumnin** [comment]
I have a mobile account whose contents copied are with my neighbours having pc or laptop. Which form the main account and the machine account?
